### ISSUE TITLE
I am getting an error with every supported file type

### ISSUE THREAD

**Issue by cobekone** [mooflu/WebViewPlus#1]
I am getting the ERR_FILE_NOT_FOUND "File can not be found - Maybe it was moved, edited or deleted." (rough translation). 

![2024-02-06 11_22_54-Window](https://github.com/mooflu/WebViewPlus/assets/64092695/2279bed1-33e1-43c6-a256-3e94d0184ac5)

![2024-02-06 11_23_36-try html](https://github.com/mooflu/WebViewPlus/assets/64092695/7e547967-7bb8-40ad-843b-eb34ece03454)

**Comment by mooflu** [comment]
Not clear how you are running it. Via quicklook plugin? What's "try.html"? Does the demo https://mooflu.github.io/WebViewPlus/ work?

**Comment by cobekone** [comment]
Yes, I am using it with the quicklook plugin. The demo works. I suspect some windows permission related thing, but other plugins are running fine. My windows account does not have administrator rights (company computer).

**Comment by mooflu** [comment]
Doesn't sound like a bug with the web app, which is what this repo is about. The actual [plugin](https://github.com/mooflu/QuickLook.Plugin.WebViewPlus) can't seem to find this app - from your 1st screenshot the web app's index.html. It should be under QuickLook\QuickLook.Plugin\QuickLook.Plugin.WebViewPlus\webviewplus - see https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions.

**Comment by mooflu** [comment]
Closing this and tracking here: https://github.com/mooflu/QuickLook.Plugin.WebViewPlus/issues/22
